src/ngRoute/route.js

```diff
@@ -615,7 +615,13 @@ function $RouteProvider() {
             }
           }, function(error) {
             if (nextRoute == $route.current) {
-              $rootScope.$broadcast('$routeChangeError', nextRoute, lastRoute, error);
+              // If nothing's listening for a $routeChangeError event, we log it in the console
+              //  to aid in debugging
+              if ($rootScope.$$listenerCount['$routeChangeError'] === 0) {
+                console.log('No listeners found for $routeChangeError: '+ error);
```

Comment:
I also feel that the error should always be logged, not only if there are zero listeners. It's just for debugging anyway.